.github/workflows/lint-workflows.yaml

```diff
@@ -5,8 +5,8 @@ on:
   pull_request: {}
```

Comment:
I think we can remove this one. The WF has been disabled for a long time https://github.com/cilium/cilium/actions/workflows/lint-codeql.yaml